initial/src/main/java/com/example/springboot/HelloController.java

```diff
@@ -1,14 +1,44 @@
 package com.example.springboot;
 
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.RequestMapping;
+import java.util.Random;
 
 @RestController
 public class HelloController {
 
+	public static final int upperbound = 25;
```

Comment:
Normally, when you declare a final variable at class level, try using capital cases all the way and split words by `_`, e.g.
`public static final int UPPER_BOUND = 25`

Reply:
noted & updated!

Reply:
noted & done!